mathesar_ui/src/sections/table-view/constraints/TableConstraint.svelte

```diff
@@ -0,0 +1,103 @@
+<script lang='ts'>
+  import { fly } from 'svelte/transition';
+  import type { FlyParams } from 'svelte/transition';
+  import {
+    Icon,
+    Seesaw,
+    Button,
+    Spinner,
+  } from '@mathesar-components';
+  import type { Constraint } from '@mathesar/stores/table-data/constraints';
```

Comment:
We can import types from the type definition export file to maintain consistency.

```suggestion
  import type { Constraint } from '@mathesar/stores/table-data/types';
```

Reply:
Addressed in 284a5a099123040c2b7e75c100a947033ed24ead